core-comments-fe/src/core-comments/resources/comments/TableList.tsx

```diff
@@ -31,7 +31,10 @@ const commentFilters = [
   <DateInput source="created_at_gte" label="After" />,
   <ReferenceInput alwaysOn label="Author" source="author_id" reference="User" perPage={10}>
```

Comment:
can't assume the reference is a User. This is a polymorphic ref

Reply:
WIP